ui/eureka/eureka.ts

```diff
@@ -760,8 +760,14 @@ class EurekaTracker {
     for (const log of e.detail.logs) {
       const flagRegex = this.TransObjectByParserLang(this.options.Regex, 'gFlagRegex');
       let match = flagRegex.exec(log);
-      if (match && match[1] && match[2] && match[3] && match[4])
-        this.AddFlag(parseFloat(match[2]), parseFloat(match[3]), match[1], match[4]);
+      if (match && match[2] && match[3]) {
+        this.AddFlag(
+          parseFloat(match[2]),
+          parseFloat(match[3]),
+          match[1] as string,
+          match[4] as string,
```

Comment:
Mostly we don't use `as`-cast

```suggestion
          match[1] ?? '',
          match[4] ?? '',
```

If you are not familiar with TypeScript or JavaScript:
If the symbol value before `??` is `undefined` or `null` then use the symbol after it.
in other words, `A ?? B` is equal to `(A === undefined || A === null) ? B : A`

Reply:
Also, you can run `npm run lintfix` after you do the changes, ESLint would try its best to automatically fix lint errors.

Reply:
Thanks! I was initially just doing this through the github text editor 😅 but I did end up using --fix for the third attempt..